DigitalOffice/Services/ProjectService/Validators/EditProjectValidator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.ProjectService.Models;
+
+namespace LT.DigitalOffice.ProjectService.Validators
+{
+    public class EditProjectValidator : AbstractValidator<EditProjectRequest>
+    {
+        public EditProjectValidator()
+        {
+            RuleFor(project => project.Name)
+                    .NotEmpty()
+                    .WithMessage("Project name is empty.")
```

Comment:
We agreed not to write this

Reply:
Well, previous validator contained this.

Fixed